augur/io.py

```diff
@@ -194,3 +195,7 @@ def write_sequences(sequences, path_or_buffer, format="fasta"):
         )
 
     return sequences_written
+
+
+def print_err(*args):
+    print(*args, file=sys.stderr)
```

Comment:
So many variations of this. :-) Still amazed Python doesn't ship with the equivalent as `warn()` or something (like C, Perl, etc).

Reply:
Yeah, custom function isn't ideal but at least it reduces updates down to one place if we want to change anything in the future.